meteor/server/api/templates/templates.ts

```diff
@@ -70,6 +70,9 @@ export type SegmentLineItemOptional = Fix<SegmentLineItem>
 export type SegmentLineItemOptional = Optional<SegmentLineItem>
 export type SegmentLineAdLibItemOptional = Optional<SegmentLineAdLibItem>
 export type RunningOrderBaselineItemOptional = Optional<RunningOrderBaselineItem>
+export type TemplateRuntimeArguments = {
```

Comment:
this looks like it should be an _interface_ instead of a _type alias_
https://medium.com/@martin_hotell/interface-vs-type-alias-in-typescript-2-7-2a8f1777af4c

Reply:
Yeah, good point!